### PR TITLE
Confirm Completion section on Review screen

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#13960

*Description of changes:*
- Added Confirm Completion component
- Validation is triggered after Continue button clicked
- Small refactor for Authorization validation
- Add Attention Reference Number to the store

### Screenshots

Screen Shot 2022-10-30 at 2 20 55 PM

Screen Shot 2022-10-30 at 2 21 02 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  name: 'ConfirmCompletion',
+
+  props: {
+    legalName: {
+      type: String,
+      required: true
+    }
+  },
+  emits: ['confirmCompletion'],
+
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue]: Spaces here intentional?

**Review by cameron-eyds — APPROVED** [review]
Looks good! 👍

**Review by owillborn — APPROVED** [review]
Looks good to me

File: ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue]: Nope. Will cleanup.

**Pull request merged**